### ISSUE TITLE
question: will this work with adam too?

### ISSUE THREAD

**Issue by romanoss** [simon-larsson/keras-swa#5]

**Comment by simon-larsson** [comment]
Yes, it works with adam too. If you use one of the learning rate schedules it can interfere with the learning rate decay in adam, but SWA will still work.
